pubspec.yaml

```diff
@@ -0,0 +1,11 @@
+name: lints
+description: Lints that represent the Dart style guide
+version: 0.1.0
+homepage: https://dart.dev/guides/language/effective-dart
+
+environment:
+  sdk: '>=2.12.0 <3.0.0'
+
+# dependencies:
+# dev_dependencies:
+# NOTE: Code is not allowed in this package. Do not add dependencies.
```

Comment:
We should file an issue in the repo to enforce this via CI.

Reply:
https://github.com/dart-lang/lints/issues/2

Reply:
Happy to take this on. 👍